latest.py

```diff
@@ -83,8 +83,9 @@ def main():
     fork_motor_group.spin_to_position(1800, DEGREES, wait=False)
     stage = "Pick up red goal 1"
 
-    goto( -1020, 1320, 0)
-    goto(  -920, 1220, 0)
+#    goto( -1020, 1320, 0)
+#    goto(  -920, 1220, 0)
+    goto(  -920, 1320, 0)
```

Comment:
Direct to -920, 1320 might save time, since each 45 degrees direction change stops the robot